mipengine/node_tasks_DTOs.py

```diff
@@ -69,24 +68,44 @@ class TableInfo(ImmutableBaseModel):
     _validate_identifier = validator("name", allow_reuse=True)(validate_identifier)
 
 
-class TableView(ImmutableBaseModel):
-    datasets: List[str]
-    columns: List[str]
-    filter: Dict
+class _ColumnData(ImmutableBaseModel):
```

Comment:
why redefining this, instead of moving the single implementation somewhere that can be used by every module that needs it?

Reply:
I agree. We have two identical definitions. I think there should be only one named TabularData.